crates/iota-replay/src/tests.rs

```diff
@@ -30,7 +29,8 @@ async fn verify_latest_tx_replay_impl() {
     let urls: Vec<_> = default_cfg
         .base_network_configs
         .iter()
-        .filter(|q| q.name != "devnet") // Devnet is not always stable
+        // TODO: enable this when mainnet is launched
+        .filter(|q| q.name != "devnet" && q.name != "mainnet") // Devnet is not always stable, mainnet is not ready
         .map(|c| c.public_full_node.clone())
         .collect();
 
```

Comment:
Btw not related to this change, but further below is a function called `extract_one_system_tx`, shouldn't this be called `extract_one_no_system_tx` instead? Fine to open an issue and addressing it separated, just want to have confirmation that this is wrong

Reply:
yeah, I think name of methods is quite wrong, because inside we don’t look at system txs
` txs.retain(|q| *q != TransactionDigest::genesis_marker());`
I can rename this function in scope of this task